frontend/.env.development

```diff
@@ -1,3 +1,3 @@
-VUE_APP_FRONTEND=http://127.0.0.1:8080
-VUE_APP_FUNCTION_KEY=SteUyT0o2YSt4JW8k2qZ5ShTBcQYQj1pxE8cO7oTrYI4ps92/53o9w==
-VUE_APP_BACKEND=https://www.theurlist.com
+VUE_APP_FRONTEND=http://127.0.0.1:8080
+VUE_APP_FUNCTION_KEY=SteUyT0o2YSt4JW8k2qZ5ShTBcQYQj1pxE8cO7oTrYI4ps92/53o9w==

```

Comment:
I know this value is probably not valid anymore, but should still .gitignore this file, similar to appsettings.development.json in the backend code

Reply:
Will keep this out for not to reduce further complicating this PR.